src/volume.ts

```diff
@@ -58,9 +58,9 @@ import {
   getWriteSyncArgs,
   unixify,
 } from './node/util';
-import type { PathLike, symlink } from 'fs';
+import type { symlink } from 'fs';
+import type { PathLike } from './node/types/misc';
```

Comment:
we're re-exporting `symlink` so we can merge these

```suggestion
import type { PathLike, symlink } from './node/types/misc';
```